scripts/validations/Readme.md

```diff
@@ -0,0 +1,21 @@
+ # opisac jak dzialac:
+* Skrypt ma na celu sprawdzenie czy uzytkownik podal wszystkie wymagane pola oraz czy odpowiadaja swojej wartosci.
+
+
+## cele co to ma robic:
+* pola nie moge byc puste
+* pola maja odpowiadac wlasciwej wartosci (email ma byc podany jako email, name jako name, number jako number).
```

Comment:
to jest niepoprawnie sformulowane:

Kazde pola formularza musi posiadac poprawny typ danych (email, number, string)